controllers/user.server.controller.ts

```diff
@@ -39,16 +40,22 @@ export async function userCreate(
     displayName: Joi.string().min(3).max(30).required(),
     email: Joi.string().email().required(),
     plaintextPassword: validators.password().required(),
+    profilePicture: Joi.string(),
   });
 
-  const formData = validate(rules, req.body);
+  const requestInfo = {
+    ...req.body,
+    profilePicture: req.file.filename,
```

Comment:
would be ideal to use the whole url rather than relative to static endpoint. If not easy dw